CRITICAL - 2022-11-28 19:35:57 --> Too few arguments to function App\Controllers\Capacitacion::index(), 0 passed in C:\laragon\www\itson\system\CodeIgniter.php on line 927 and exactly 1 expected
in APPPATH\Controllers\Capacitacion.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->index()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:38:09 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:38:09 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:39:54 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:39:54 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:41:49 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:41:49 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:41:57 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:41:57 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:42:28 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:42:28 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:45:26 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:45:26 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:46:03 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:46:03 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:47:36 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:47:36 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:48:45 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:48:45 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:49:34 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:49:34 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:51:50 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:51:50 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:54:41 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:54:41 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:55:04 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:55:04 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-28 19:58:43 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\laragon\www\itson\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\itson\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\itson\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\itson\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\itson\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\itson\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\itson\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\itson\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\itson\app\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\itson\system\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
#9 C:\laragon\www\itson\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
#10 C:\laragon\www\itson\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\itson\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-11-28 19:58:43 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `capacitacion` (`nombre`, `tipo`, `lugar`, `fecha_cursada`, `institucion`, `modalidad`, `duracion_horas`, `evidencia`, `id_usuario`) VALUES (:nombre:, :tipo:, :lugar:, :fecha_cursada:, :institucion:, :modalidad:, :duracion_horas:, :evidencia:, :id_usuario:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Capacitacion.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Capacitacion->save()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Capacitacion))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 20:51:04 --> Invalid file: form_usuario.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('form_usuario.php')
 2 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('form_usuario', [], true)
 3 APPPATH\Controllers\Usuario.php(96): view('form_usuario')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->form()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 20:51:38 --> Too few arguments to function view(), 0 passed in C:\laragon\www\itson\app\Controllers\Usuario.php on line 96 and at least 1 expected
in SYSTEMPATH\Common.php on line 1120.
 1 APPPATH\Controllers\Usuario.php(96): view()
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->form_usuario()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:01:44 --> syntax error, unexpected '$password' (T_VARIABLE)
in APPPATH\Controllers\Usuario.php on line 97.
 1 SYSTEMPATH\Router\Router.php(499): CodeIgniter\Router\AutoRouter->getRoute('Usuario/login')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('Usuario/login')
 3 SYSTEMPATH\CodeIgniter.php(802): CodeIgniter\Router\Router->handle('Usuario/login')
 4 SYSTEMPATH\CodeIgniter.php(430): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:02:00 --> syntax error, unexpected '}', expecting ';'
in APPPATH\Controllers\Usuario.php on line 103.
 1 SYSTEMPATH\Router\Router.php(499): CodeIgniter\Router\AutoRouter->getRoute('Usuario/login')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('Usuario/login')
 3 SYSTEMPATH\CodeIgniter.php(802): CodeIgniter\Router\Router->handle('Usuario/login')
 4 SYSTEMPATH\CodeIgniter.php(430): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:03:10 --> Too few arguments to function App\Controllers\Usuario::iniciarSesion(), 0 passed in C:\laragon\www\itson\system\CodeIgniter.php on line 927 and exactly 2 expected
in APPPATH\Controllers\Usuario.php on line 94.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:03:35 --> Too few arguments to function App\Controllers\Usuario::iniciarSesion(), 0 passed in C:\laragon\www\itson\system\CodeIgniter.php on line 927 and exactly 2 expected
in APPPATH\Controllers\Usuario.php on line 94.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:04:29 --> Too few arguments to function App\Controllers\Usuario::iniciarSesion(), 0 passed in C:\laragon\www\itson\system\CodeIgniter.php on line 927 and exactly 2 expected
in APPPATH\Controllers\Usuario.php on line 94.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:04:38 --> Too few arguments to function App\Controllers\Usuario::iniciarSesion(), 0 passed in C:\laragon\www\itson\system\CodeIgniter.php on line 927 and exactly 2 expected
in APPPATH\Controllers\Usuario.php on line 94.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:05:00 --> Undefined variable: capacitaciones
in APPPATH\Views\Capacitacion\index.php on line 66.
 1 APPPATH\Views\Capacitacion\index.php(66): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: capacitaciones', 'APPPATH\\Views\\Capacitacion\\index.php', 66, [])
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\Capacitacion\\index.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('Capacitacion/index', [], true)
 5 APPPATH\Controllers\Usuario.php(100): view('Capacitacion/index')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion('00000229115', 'fasadad')
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:05:23 --> Undefined variable: capacitaciones
in APPPATH\Views\Capacitacion\index.php on line 66.
 1 APPPATH\Views\Capacitacion\index.php(66): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: capacitaciones', 'APPPATH\\Views\\Capacitacion\\index.php', 66, [])
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\Capacitacion\\index.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('Capacitacion/index', [], true)
 5 APPPATH\Controllers\Usuario.php(100): view('Capacitacion/index')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion('00000229115', 'fsaasf')
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:11:42 --> Too few arguments to function App\Controllers\Usuario::iniciarSesion(), 0 passed in C:\laragon\www\itson\system\CodeIgniter.php on line 927 and exactly 2 expected
in APPPATH\Controllers\Usuario.php on line 94.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:13:07 --> The route for "capacitaciones/1" cannot be found.
in SYSTEMPATH\HTTP\RedirectResponse.php on line 60.
 1 SYSTEMPATH\HTTP\RedirectResponse.php(60): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('capacitaciones/1')
 2 APPPATH\Controllers\Usuario.php(100): CodeIgniter\HTTP\RedirectResponse->route(false)
 3 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion('00000229115', 'sdadad')
 4 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:13:28 --> The route for "capacitaciones" cannot be found.
in SYSTEMPATH\HTTP\RedirectResponse.php on line 60.
 1 SYSTEMPATH\HTTP\RedirectResponse.php(60): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('capacitaciones')
 2 APPPATH\Controllers\Usuario.php(100): CodeIgniter\HTTP\RedirectResponse->route(false)
 3 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion('00000229115', 'sdadad')
 4 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:14:34 --> syntax error, unexpected ';', expecting ')'
in APPPATH\Controllers\Usuario.php on line 100.
 1 SYSTEMPATH\Router\Router.php(499): CodeIgniter\Router\AutoRouter->getRoute('Usuario/iniciarSesion/00000229115/sdadad')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('Usuario/iniciarSesion/00000229115/sdadad')
 3 SYSTEMPATH\CodeIgniter.php(802): CodeIgniter\Router\Router->handle('Usuario/iniciarSesion/00000229115/sdadad')
 4 SYSTEMPATH\CodeIgniter.php(430): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:14:47 --> syntax error, unexpected ''/Capacitacion/index/'' (T_CONSTANT_ENCAPSED_STRING), expecting ')'
in APPPATH\Controllers\Usuario.php on line 100.
 1 SYSTEMPATH\Router\Router.php(499): CodeIgniter\Router\AutoRouter->getRoute('Usuario/iniciarSesion/00000229115/sdadad')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('Usuario/iniciarSesion/00000229115/sdadad')
 3 SYSTEMPATH\CodeIgniter.php(802): CodeIgniter\Router\Router->handle('Usuario/iniciarSesion/00000229115/sdadad')
 4 SYSTEMPATH\CodeIgniter.php(430): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:15:06 --> syntax error, unexpected ''/Capacitacion/index/1'' (T_CONSTANT_ENCAPSED_STRING), expecting ')'
in APPPATH\Controllers\Usuario.php on line 100.
 1 SYSTEMPATH\Router\Router.php(499): CodeIgniter\Router\AutoRouter->getRoute('Usuario/iniciarSesion/00000229115/sdadad')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('Usuario/iniciarSesion/00000229115/sdadad')
 3 SYSTEMPATH\CodeIgniter.php(802): CodeIgniter\Router\Router->handle('Usuario/iniciarSesion/00000229115/sdadad')
 4 SYSTEMPATH\CodeIgniter.php(430): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:16:03 --> The route for "/Capacitacion/index/1" cannot be found.
in SYSTEMPATH\HTTP\RedirectResponse.php on line 60.
 1 SYSTEMPATH\HTTP\RedirectResponse.php(60): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('/Capacitacion/index/1')
 2 SYSTEMPATH\Common.php(860): CodeIgniter\HTTP\RedirectResponse->route(false)
 3 APPPATH\Controllers\Usuario.php(100): redirect('/Capacitacion/index/1')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion('00000229115', 'sdadad')
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:16:05 --> The route for "/Capacitacion/index/1" cannot be found.
in SYSTEMPATH\HTTP\RedirectResponse.php on line 60.
 1 SYSTEMPATH\HTTP\RedirectResponse.php(60): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('/Capacitacion/index/1')
 2 SYSTEMPATH\Common.php(860): CodeIgniter\HTTP\RedirectResponse->route(false)
 3 APPPATH\Controllers\Usuario.php(100): redirect('/Capacitacion/index/1')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion('00000229115', 'sdadad')
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:19:27 --> The route for "/Capacitacion/index/1" cannot be found.
in SYSTEMPATH\HTTP\RedirectResponse.php on line 60.
 1 SYSTEMPATH\HTTP\RedirectResponse.php(60): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('/Capacitacion/index/1')
 2 SYSTEMPATH\Common.php(860): CodeIgniter\HTTP\RedirectResponse->route(false)
 3 APPPATH\Controllers\Usuario.php(100): redirect('/Capacitacion/index/1')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion('0000222', '2123')
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-28 21:20:09 --> Too few arguments to function App\Controllers\Usuario::iniciarSesion(), 0 passed in C:\laragon\www\itson\system\CodeIgniter.php on line 927 and exactly 2 expected
in APPPATH\Controllers\Usuario.php on line 94.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Usuario->iniciarSesion()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
